src/main/resources/files/readme.md.secondary.tpl

```diff
@@ -36,6 +36,10 @@ However, any method to send a REST request with an appropriate header will work
 
 
 [# th:if="${mp_rest_client}"]
+### Demo Execution
+
+Once the service-a server is running (see `demo/service-a/readme.md`) and the service-b server is running, open your browser at the following URL to launch the test page and execute both projects: http://localhost:[# th:text="${port_service_a}"/]/index.html
```

Comment:
This info should be in readme.md.top.tpl.

Reply:
Since you have already added to service-a, you can just remove the lines 39-41.

Reply:
Agreed.  Done.